kubetest/extract_k8s.go

```diff
@@ -442,7 +442,7 @@ func (e extractStrategy) Extract(project, zone, region string, extractSrc bool)
 			if err != nil {
 				return fmt.Errorf("failed to get latest gke version: %s", err)
 			}
-			return getKube("https://storage.googleapis.com/kubernetes-release-gke/release", version, extractSrc)
+			return getKube("https://storage.googleapis.com/gke-release/release", version, extractSrc)
```

Comment:
It seems broken.
A) I would expect that URL to contain kubernetes part: `https://storage.googleapis.com/gke-release/kubernetes/release`
B) Still it will not work for testing GKE on staging env as the version is put `gke-release` bucket only after promotion. Staging uses `gke-release-staging`. It seems like we could always use `gke-release-staging` as it should be strict superset of  `gke-release`. I am not sure if permissions will be fine though.